### PR TITLE
Floating Debris Effect

### PR DESCRIPTION
**Pull request by PangwenE**
idk wtf to put here

### CODE REVIEW

File: src/notes/floatingdebris.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                dupe.animation._definitePosition = [[0, 0, 0, 0], [0, 0, 0, 0.5], [Random(-3, 3), Random(-3, 3), 5, 1, "easeOutCubic"]];
+            }
+        
+            dupe.push();
+        }
+        
+    });
+};
+
+// supermemer is a furry
```
Comment (IMB11) [inline src/notes/floatingdebris.ts]:
```suggestion
```

File: src/index.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ export * from './decor/water'
 
 export * from './notes/ghosty'
 export * from './notes/notetrail'
+export * from './notes/floatingdebris'
```
Comment (cal117) [inline src/index.ts:15]: Seems good 👍

**Review by IMB11 — CHANGES_REQUESTED** [review]

**Comment by IMB11** [comment]
@StormPacer is checking functionality

File: src/notes/floatingdebris.ts (hunk 1 above)
Comment (PangwenE) [reply src/notes/floatingdebris.ts]: 😒

**Review by IMB11 — APPROVED** [review]

**Comment by StormPacer** [comment]
The effect works, approving.

**Pull request merged**